Update 18-working-with-marketing.rst (#1168)
writethedocs/www: docs/book-club/splunk-product-docs/18-working-with-marketing.rst

===================================
Chapter 18: Working with Marketing
===================================

Summary of Chapter
------------------
This chapter discussed how technical writers can & should work with the Marketing department. It raises the question of docs' role: informative, sales-driven, or both?

> Docs can serve as marketing material, though they should aim for educating the user first & act as a sales pitch second. Marketing may not like the direct nature of docs, but it is crucial to maintain honesty & directness.

Reviewer: ```suggestion
Docs can serve as marketing material, though they should aim for educating the user first and act as a sales pitch second. Marketing may not like the direct nature of docs, but it is crucial to maintain honesty and directness.
```

> Personas are a key element to both Tech Writers and Marketing. These do not always align though, as Marketing personas target buyers (Management), whereas docs target users (end users/developers).

Reviewer: ```suggestion
Personas are a key element to both tech writers and marketing. These do not always align though, as marketing personas target buyers (management), whereas docs target users (end users/developers).
```

> How can Marketing help Docs:

Reviewer: ```suggestion
How can marketing help docs:
```

* Ensure the doc team knows marketing priorities and feature naming conventions
* Share customer feedback and perspective
* Understand the market players and how competitors document their products
* Align on customer goals
* Understand future goals/plans

> How Docs can Help Marketing:

Reviewer: ```suggestion
How docs can help marketing:
```

* Share your knowledge of what users find helpful
* Information you develop could inform marketing content

> * Your product knowledge can improve the relevance of Marketing content

Reviewer: ```suggestion
* Your product knowledge can improve the relevance of marketing content
```

> Get & stay in contact with Marketing. Working together helps tighten the link between your docs & marketing content, avoids confusion for the user, and can make for a sharper content experience from interest to implementation.

Reviewer: ```suggestion
Get and stay in contact with marketing. Working together helps tighten the link between your docs and marketing content, avoids confusion for the user, and can make for a sharper content experience from interest to implementation.
```

> Key Takeaways from Discussion

Reviewer: ```suggestion
Key takeaways from discussion
```

-----------------------------

> A big challenge: push-back from Marketing to make content more sales-y.

Reviewer: ```suggestion
A big challenge: push-back from marketing to make content more sales-y.
```

> Both tech writers & marketing aim to help a user achieve a goal: Selling the product that enables the user to achieve said goal (Marketing), and educating the user on how to use the product in this manner (Tech Writer).

Reviewer: ```suggestion
Both tech writers and marketing aim to help a user achieve a goal: selling the product that enables the user to achieve said goal (marketing), and educating the user on how to use the product in this manner (tech writer).
```

> Tech Writers can share the following with Marketing:

Reviewer: ```suggestion
Tech writers can share the following with marketing:
```

* Expertise on the product.
* Tutorials based on Marketing's sales points.
* Easy links to docs, so they can point users to the appropriate material

> Marketing can share the following with Tech Writers:

Reviewer: ```suggestion
Marketing can share the following with tech writers:
```

> * Communications advice when you have to share bad news (forced plan changes/migrations, dropping a feature, etc.).

Reviewer: ```suggestion
* Communications advice when you have to share bad news (such as forced plan changes/migrations or dropping a feature).
```

* Assistance with ensuring the places that hold my content are "on brand."
* Design tips for screenshots and templates.

Explain features and benefits to them in ways that they find more understandable than when the developers or engineers do the explaining. 
Also, point out when the marketing docs aren't technically accurate or are missing some major technical benefits.

> Use the same language to describe products, and how to use them. May require differences in tone between docs & marketing, but aim to keep the user experience as similar as possible.

Reviewer: ```suggestion
Use the same language to describe products, and how to use them. May require differences in tone between docs and marketing, but aim to keep the user experience as similar as possible.
```
